test/windows/local_test.rb

```diff
@@ -40,6 +40,33 @@
     cmd.stderr.must_equal ''
   end
 
+  it 'uses a named pipe if available' do
+    # Must call `:conn` early so we can stub `SecureRandom`
+    connection = conn
+
+    SecureRandom.stubs(:hex).returns('with_pipe')
+    cmd = connection.run_command('Get-ChildItem //./pipe/ | Where-Object { $_.Name -Match "inspec_with_pipe" }')
+    cmd.stdout.wont_be_nil
+    cmd.stderr.must_equal ''
+  end
+
+  it 'when named pipe is not available it runs `Mixlib::Shellout`' do
```

Comment:
I don't see anything obvious in this test that actually tests that Mixlib::ShellOut is used instead of the named pipe. How are you asserting that in this test?

Reply:
Great point, I added words to the comments to make it more clear. Basically, it runs a PowerShell command and that command checks to see if a pipe exists. If it does then we know it used the pipe and not `Mixlib::ShellOut`.